CHANGES.rst

```diff
@@ -1,6 +1,18 @@
 Changes
 =======
 
+Version 0.6.0 (2019-12-20)
+--------------------------
+
+- Adds user secrets management mechanism.
+- Adds `--workflow` flag for specifying workflow for `run` and `create`
+  commands.
+- Adds version command.
+- Modifies file upload to use `requests` instead of Bravado.
```

Comment:
Since this should be readable by final users, I would say something more along these lines, let me know what you think:
More positive:
```
Improves file uploads by increasing reliability when sending big files.
```
or less positive but more straight forward:
```
Fixes big file uploads problem.
```